modules/gitops-release-notes-1-6-2.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-6-2_{context}"]
+= Release notes for {gitops-title} 1.6.2
+
+{gitops-title} 1.6.2 is now available on {product-title} 4.8, 4.9, 4.10 and 4.11.
+
+[id="fixed-issues-1-6-2_{context}"]
+== Fixed issues
+
+The following issue have been resolved in the current release:
+
+* Before this update, subscription health check was marked *degraded* for missing *InstallPlan* when more than 5 operators are installed in a project. This update fixes the issue. link:https://issues.redhat.com/browse/GITOPS-2018[GITOPS-2018]
+
+* Before this update, the GitOps Operator would spam the cluster with deprecation notice warning whenever it detects that an Argo CD instance is using deprecated fields. This update fixes this the issue and only one warning event is shown for each detected field, for each instance that specifies them. link:https://issues.redhat.com/browse/GITOPS-2230[GITOPS-2230]
```

Comment:
Again with the past and present tense. Also the last sentence was worded in such a way that I had to read it several times. What do you think of my rephrase. Do you think it makes equal sense? 

Before this update, the GitOps Operator would spam the cluster with deprecation notice warning whenever it _detected_ that an Argo CD instance _used_ deprecated fields. This update fixes this issue and shows only one warning event for each instance that detects a field.